default.nix

```diff
@@ -39,7 +39,11 @@ with holonix.pkgs;
     ''
     ];
 
-  buildInputs = [ holonix.pkgs.libiconv ]
+  # Needed so bindgen can find libclang.so
+  LIBCLANG_PATH="${holonix.pkgs.llvmPackages.libclang}/lib";
+  hardeningDisable = [ "fortify" ];
+
+  buildInputs = [ holonix.pkgs.libiconv holonix.pkgs.pcre holonix.pkgs.cmake holonix.pkgs.clang]
```

Comment:
still compiles after removing gblic @thedavidmeister

Reply:
@AshantiMutinta can we remove all the extra build inputs?

i thought that with `LIBCLANG_PATH` and `hardeningDisable` it should all work without this change here?

Reply:
not sure, will try but I remember us incrementally adding these build inputs., had to add pcre cause it was looking for it in the nix-shell but we probably don't need cmake

Reply:
Removed the build inputs

Reply:
pcre is needed so restored it to the build inputs